packages/cli/test/commands/upgrade.test.ts

```diff
@@ -51,44 +53,66 @@ describe('upgrade:command', async () => {
 
     const report: Report = readJSONSync(reportFile);
 
-    expect(report).toHaveProperty('summary');
-
-    expect(report.summary.projectName).toBe('@rehearsal/cli');
-    expect(report.summary.tsVersion).toBe(latestPublishedTSVersion);
-    expect(report.summary.uniqueErrors).toBe(3);
-    expect(report.summary.totalErrors).toBe(25);
-    expect(report.summary.totalErrorsList).toStrictEqual({
-      '2322': 1,
-      '2616': 3,
-      '6133': 21, 
-    });
-    expect(report.summary.fixedErrors).toBe(0);
-    expect(report.summary.fixedErrorsList).toStrictEqual({
-      '2322': 0,
-      '2616': 0,
-      '6133': 0,
-    });
-    expect(report.summary.files).toBe(3);
-    expect(report.summary.filesList).toStrictEqual([
-      '/foo/foo.ts',
-      '/foo_2/foo_2a.ts',
-      '/foo_2/foo_2b.ts',
+    expect(report).toMatchSnapshot();
+  });
+});
+
+describe('upgrade:command typescript@rc', async () => {
+  beforeAll(beforeSetup);
+  afterAll(afterEachCleanup);
+
+  test('runs', async () => {
+    const buildTag = 'rc';
+
+    const result = await runTSNode('upgrade', [
+      FIXTURE_APP_PATH,
+      '--report',
+      'json',
+      '--dryRun',
+      '--build',
+      buildTag,
     ]);
-    expect(report.items.length).toBe(25);
 
-    const firstFileReportError = report.items[0];
+    //4.8.1-rc
+    const latestPublishedTSVersion = await getLatestTSVersion(buildTag);
+    const reportFile = join(FIXTURE_APP_PATH, '.rehearsal', 'report.json');
 
-    expect(firstFileReportError.errorCode).toEqual(2616);
-    expect(firstFileReportError.category).toEqual('Error');
-    expect(firstFileReportError.fixed).toEqual(false);
-    expect(firstFileReportError.nodeKind).toEqual('ImportSpecifier');
-    expect(firstFileReportError.nodeText).toEqual('execa');
-    expect(firstFileReportError.message).toEqual(
-      `'execa' can only be imported by using 'import execa = require("execa")' or a default import.`
-    );
-    expect(firstFileReportError.hint).toEqual(
-      `'execa' can only be imported by using 'import execa = require("execa")' or a default import.`
-    );
+    expect(result.stdout).contain(`Rehearsing with typescript@${latestPublishedTSVersion}`);
+    expect(result.stdout).to.contain(`Codefixes applied successfully`);
+    expect(existsSync(reportFile)).toBeTruthy;
+
+    const report: Report = readJSONSync(reportFile);
+
+    expect(report).toMatchSnapshot();
+  });
+});
+
+describe('upgrade:command typescript@latest', async () => {
+  beforeAll(beforeSetup);
+  afterAll(afterEachCleanup);
+
+  test('runs', async () => {
+    const buildTag = 'latest';
+
+    const result = await runTSNode('upgrade', [
+      FIXTURE_APP_PATH,
+      '--report',
+      'json',
+      '--dryRun',
+      '--build',
+      buildTag,
+    ]);
+    // 4.8.4
```

Comment:
Comment will retire soon :)

Reply:
the comment is just an example of the response
eg. `// 4.8.4`